pkg/cmd/roachtest/test_runner.go

```diff
@@ -1265,11 +1265,16 @@ func (r *testRunner) runTest(
 				}
 
 				output := fmt.Sprintf("%s\ntest artifacts and logs in: %s", failureMsg, t.ArtifactsDir())
-				params := getTestParameters(t, issueInfo.cluster, issueInfo.vmCreateOpts)
+				// githubMsg may contain additional info not needed in output
```

Comment:
Nit: the "not needed" modifier is confusing; the two outputs are mutually disjoint. Perhaps renaming `output` to `stdoutMsg` would suffice to disambiguate?